yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -156,87 +185,65 @@ func packAndSendAssessMigrationPayload(status string, errMsg string) {
 		}),
 	}
 
-	unsupportedDatatypesList := lo.Map(assessmentReport.UnsupportedDataTypes, func(datatype utils.TableColumnsDataTypes, _ int) string {
-		return datatype.DataType
-	})
+	explanation, err := buildMigrationComplexityExplanation(source.DBType, assessmentReport, "")
+	if err != nil {
+		log.Errorf("failed to build migration complexity explanation: %v", err)
+	}
+
+	var obfuscatedIssues []callhome.AssessmentIssueCallhome
+	for _, issue := range assessmentReport.Issues {
+		obfuscatedIssue := callhome.AssessmentIssueCallhome{
+			Category:            issue.Category,
+			CategoryDescription: issue.CategoryDescription,
+			Type:                issue.Type,
+			Name:                issue.Name,
+			Description:         issue.Description,
+			Impact:              issue.Impact,
+			ObjectType:          issue.ObjectType,
+			ObjectName:          constants.OBFUSCATE_STRING,
+		}
 
-	groupedByConstructType := lo.GroupBy(assessmentReport.UnsupportedQueryConstructs, func(q utils.UnsupportedQueryConstruct) string {
-		return q.ConstructTypeName
-	})
-	countByConstructType := lo.MapValues(groupedByConstructType, func(constructs []utils.UnsupportedQueryConstruct, _ string) int {
-		return len(constructs)
-	})
+		// allowing object name for unsupported extension issue type
+		if issue.Type == UNSUPPORTED_EXTENSION_ISSUE_TYPE {
```

Comment:
As part of https://yugabyte.atlassian.net/browse/DB-14476, we had decided to create separate  items for each extension type. Let's keep that behavior as is. Shall we modify the `Type` field to also include the extension name?